### PR TITLE
initial version of scale test tools

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=scale testcase=ScaleTest#testScaleTestToolsWork test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=scale testcase=ScaleTest#testScaleTestToolsWork test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=scale testcase=ScaleTest#testScaleTestToolsWork test_only

**Comment by famartinrh** [comment]
@enmasse-ci run tests profile=scale testcase=ScaleTest#testScaleTestToolsWork test_only

File: systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java [hunk 1]
```diff
@@ -191,5 +258,36 @@ private void setupEnv() throws Exception {
...
     }
+
+    List<Address> createInitialAddresses(int addresses) throws Exception {
+        int operableAddresses = 0;
+        int iterator = 0;
+
+        while (operableAddresses < addresses) {
+            try {
+                getResourceManager().appendAddresses(false, getTenantAddressBatch(addressSpace).toArray(new Address[0]));
+                if (iterator % 200 == 0) {
```
Comment (kornys) [inline systemtests/src/test/java/io/enmasse/systemtest/scale/ScaleTest.java]: operable addresses are filled only if iterator mod 200 is 0, so if you specify 10 addresses you anyway get 1000 addresses

**Pull request merged**